### DISCUSSION TITLE
Is Placement blockage feature available in OpenROAD?

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD#1851] [General]
This #openlane slack channel conversation. https://skywater-pdk.slack.com/archives/C016H8WJMBR/p1652291545428459

User want to create a placement blockage to avoid placement of standard cells in that particular area.

Is that feature available?

If yes please point us command...

@maliberty fyi

**Comment by maliberty** [comment]
We support them coming from DEF.  You can use the odb api to create them in the database directly (dbBlockage_create) but we don't have a user-level command to make that nicer.
